src/tests/Register.test.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import ShallowRenderer from 'react-test-renderer/shallow';
+import { render, fireEvent, screen, waitForElement } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Register from "../register/Register";
+
+
+test('renders Register', () => {
+  const renderer = new ShallowRenderer();
```

Comment:
Would prefer if you manually checked important things you need to using React Testing Library. Would not prefer snapshot being used. With snapshots, for every small change you need to update the snapshot. And you don't really see what exactly is being tested. eg. This test just says matchSnapshot() it feels like a blackbox. We'd rather test few important things, eg. if few values are being displayed. Some of the reasons I prefer not to use snapshots - https://medium.com/@ntgard/jest-snapshot-testing-the-bad-parts-c93aca187ba5

Reply:
again, just following your example of what's been done at work. Was not aware that you are not in favour of this (since otherwise you would've mentioned it). But IMO, I think having the snapshot matching to the whole form is a good start for the basic mvp testing. If we do want to extend it to which fields must display what, we can do that toward s the end of GSoC when I have at least the mvp done. 
This is also why I asked you modify the acceptance criteria of the PR so it is clear what required tests need to be done. What I have done so far is just to follow what's been there already (testing functionalities aka successful and not successful tests)

Reply:
I would not like us to start with snapshot testing considering it turns into bad practices later. I'd rather not have this test than have it as a snapshot test. So if you like remove this test, but I'd prefer not to use snapshot. Sorry I think I've had learnings over a period of time as well, maybe that's why I forgot to mention originally.